src/App.js

```diff
@@ -1,23 +1,37 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, NavLink, Route, Switch } from 'react-router-dom';
+import TeamDetail from './views/Teams/TeamDetail';
+import TeamList from './views/Teams/TeamList';
+import Home from './views/Home/Home';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <header>
+          Sport!
+          <NavLink to="/" className="App-link" exact>
+            Home
+          </NavLink>
+          <NavLink to="/teams" className="App-link" exact>
+            Teams
+          </NavLink>
+        </header>
+        <Switch>
```

Comment:
In order to display your player information on the deployed site you'll want to make routes for it here